Core/clim-basic/repaint.lisp

```diff
@@ -232,13 +232,13 @@
                                                                                   (sheet-region sheet)))))))
     (let* ((parent (sheet-mirrored-ancestor sheet))
            (native-sheet-region (effective-repaint-region parent sheet region)))
-          (with-sheet-medium (medium parent)
-            (with-drawing-options (medium :clipping-region native-sheet-region
-                                          :ink (pane-background sheet)
-                                          :transformation +identity-transformation+)
-              (with-bounding-rectangle* (left top right bottom)
-                  native-sheet-region
-                (medium-draw-rectangle* medium left top right bottom t)))))))
+      (with-sheet-medium (medium parent)
+        (letf (((medium-clipping-region medium) native-sheet-region)
```

Comment:
It does, it is changed to avoid a forward-dependency. Nevertheless the intention here is to clear the area, not to draw something, so it is semantically correct. Whenever specialization on always-repaint-background-mixin belongs to repaint.lisp is another question (arguably that should be handled by the medium itself).